### PR TITLE
Adds 3 tests for SMB EICAR file

### PR DESCRIPTION
**Pull request by BasTNCY**
3 tests for smb EICAR file write : 
Regular write, segmentation in random size, SMBv1
Regular write, two NetBIOS messages in one TCP packet, SMBv1
Regular write, segmentation in 2 pieces after the SMB header, SMBv1

### CODE REVIEW

**Comment by BasTNCY** [comment]
CC @catenacyber

**Comment by catenacyber** [comment]
There seems to be 3 tests instead of one

File: tests/smb-eicar-file-nbss-more-ffsmb/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Description
+
+Test SMB EICAR file rule.
+
+# PCAP
+
+The pcap comes from running Linux client smbclient against a Windows 2019 Server (with a shared folder public without needed authentication)
+
+Needs a Proxy that sends two netbios requests in one segment
```
Comment (catenacyber) [inline tests/smb-eicar-file-nbss-more-ffsmb/README.md]: Do you mean two NetBIOS messages in one TCP packet ?

Comment (BasTNCY) [reply tests/smb-eicar-file-nbss-more-ffsmb/README.md]: Yes, that's it. I edited it

Comment (catenacyber) [reply tests/smb-eicar-file-nbss-more-ffsmb/README.md]: Still waiting for the `git push` or new PR

**Comment by jasonish** [comment]
Are we waiting on an update for this test?

**Comment by catenacyber** [comment]
> Are we waiting on an update for this test?

Yes just for one README file

**Comment by catenacyber** [comment]
Looks good to me now, thanks @BasTNCY

**Comment by victorjulien** [comment]
Can the last readme commit be squashed into the commit that it fixes?

**Pull request closed without merging**